afs/templates/views/components/reports/physical-thing.htm

```diff
@@ -184,21 +184,240 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
 <div class="model-summary-report">
     <div class="afs-report-page">
         <!-- Names -->
-        <div data-bind="component: {
-            name: 'views/components/reports/scenes/name',
-            params: {
-                data: resource,
-                cards: nameCards
-            }
-        }"></div>
-        <!-- Description -->
-        <div class="margin-top-summary" data-bind="component: {
-            name: 'views/components/reports/scenes/description',
-            params: {
-                data: resource,
-                cards: descriptionCards,
-            }
-        }"></div>
+        <div class="afs-report-section">
+            <div class="afs-report-subsection">
+                <!-- ko ifnot: nameSummary() && nameSummary().length -->
+                <div class="afs-nodata-note">No names for this resource</div>
+                <!-- /ko -->
+
+                <!-- ko if: nameSummary() && nameSummary().length -->
+                <div class="afs-table pad-btm">
+                    <table class="table table-striped" cellspacing="0" width="100%">
+                        <thead>
+                            <tr class="afs-table-header">
+                                <th>{% trans "Name" %}</th>
+                                <th>{% trans "Name Type" %}</th>
+                            </tr>
+                        </thead>
+                        <tbody data-bind="dataTablesForEach: { data: nameSummary, dataTableOptions: getTableConfig(2) }">
+                            <tr>
+                                <td data-bind="text: content"></td>
+                                <td data-bind="text: type"></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+                <!-- /ko -->
+            </div>
+        </div>
+        <!-- Image -->
+        <div class="afs-report-section">
```

Comment:
Can you add a header here in the same font style as the table column headers?  There's clarity in what data is displayed in the others; but because this isn't a table, it's harder to know what exactly it is.  Something like "Image Manifests" may do.